sdk/src/sysvar/instructions.rs

```diff
@@ -0,0 +1,18 @@
+//! This account contains the serialized transaction instructions
+//!
+
+use crate::instruction::Instruction;
+use crate::sysvar::Sysvar;
+
+pub type Instructions = Vec<Instruction>;
+
+crate::declare_sysvar_id!("instructions1111111111111111111111111111111", Instructions);
+
+impl Sysvar for Instructions {}
+
+pub fn get_current_instruction(data: &[u8]) -> u16 {
+    let mut instr_fixed_data = [0u8; 2];
```

Comment:
`byteorder` might also work well here

Reply:
trent and I did some benchmarking and looking at assembly in another case and basically the compiler optimizes them both down to the same thing.

Reply:
both being bincode and byteorder?

Reply:
Just comparing something like `u16.to_le_bytes()` vs. `byteorder::LittleEndian::write_u16()`

bincode seems like to make a lot of code, so I'm not sure about that one.